nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/PayaraHazelcastDiscoveryFactory.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+ *
+ * Copyright (c) [2016-2023] Payara Foundation and/or its affiliates. All rights reserved.
```

Comment:
```suggestion
 * Copyright (c) [2023] Payara Foundation and/or its affiliates. All rights reserved.
```

Brand new class shouldn't get copyright from 2016 😜

Reply:
none of this is _new_ but ok :)